testsuite/tests/conftest.py

```diff
@@ -627,7 +627,9 @@ def _resolve_rhsso(testconfig, tools, rhsso_kind):
     key = "no-ssl-rhbk"
```

Comment:
it's selecting no-ssl-rhbk, is there any discovery for ssl variant, and skip if such rhbk is not found?